.coderabbit.yaml

```
@@ -0,0 +1,17 @@
+  high_level_summary: true
+  auto_title_placeholder: "@coderabbitai"
+  commit_status: true
+  poem: true
+  review_status: true
+  collapse_walkthrough: true
+  auto_review:
+    enabled: true
+    drafts: false
+    base_branches: ["main"]
+chat:
+  auto_reply: true
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)